src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarSizeInfo.tsx

```diff
@@ -17,7 +17,10 @@ export class ArtworkSidebarSizeInfo extends Component<
     } = this.props
     if (
       !(edition_of && edition_of.length) &&
-      !(dimensions && (dimensions.in || dimensions.cm))
+      !(
+        dimensions &&
+        (dimensions?.in?.match(/\d/) || dimensions?.cm?.match(/\d/))
```

Comment:
What's the value here that's manifesting the behavior we're seeing?

Reply:
Also I wouldn't be too sure that we shouldn't display non digit string values. If these are strings there's probably a pretty reasonable reason for that. Though, I can't think of one — I feel like @anandaroop would know if there's something weird we put in here that is none-the-less valid?

Reply:
FWIW https://github.com/artsy/gravity/blob/main/app/models/util/dimensions.rb

Reply:
CMS does currently enforce numeric-only input through validations like [these](https://github.com/artsy/volt/blob/9b6cbbde427d9ce712edc9a17fda340bc43a392a/app/javascript/apps/ArtworkForm/validation/width.ts)

But I'm pretty certain this was not always the case, and that we could very well have some non-numeric chars in older artworks.

(Also `\d` might be too strict, we'd need `\.` too at least, even in the most optimistic case)

Reply:
Sure enough…

```
pp Artwork.find('4d8b92fc4eb68a1b2c000b3c').attributes.slice(:_id, :width, :height, :depth, :diameter, :created_at, :published)

{"_id"=>BSON::ObjectId('4d8b92fc4eb68a1b2c000b3c'),
 "width"=>"22 1/4",
 "height"=>"20 1/2",
 "depth"=>"",
 "created_at"=>2010-12-20 19:48:55 UTC,
 "published"=>true}
```

Not sure if we have anything more exotic than virgules though.

Reply:
Not sure I even meant to PR this - lol

I updated this with a more forgiving rule: the dimensions should include a digit somewhere.

Reply:
👍 could we add a spec too?

Reply:
@dzucconi - I think [this](https://github.com/artsy/force/pull/9191/files#diff-2657a287f0af87d5a20febe783f6ffb886fbd2c2897a83c6cba7fac96de7574bR122) should cover it, right @jacobherrington?

Reply:
It apparently wasn't covering it before though?